go/vt/vttablet/tabletserver/messager/message_manager.go

```diff
@@ -323,6 +327,7 @@ func (mm *messageManager) runSend() {
 					if mr.Epoch >= 1 {
 						lateCount++
 					}
+					totalDelay += time.Since(time.Unix(0, mr.TimeCreated))
```

Comment:
If you're using Timings, you have two options:
1. Record every `row` here along with its delay.
2. Record every `send` outside the loop by accumulating the total delay.

There's no internal difference because we only send one row per message.

Reply:
Calling Record for each row, it seems cheap.